subgraphs/apeswap/schema.graphql

```diff
@@ -0,0 +1,557 @@
+# Subgraph Schema: DEX AMM
+# Version: 1.0.1
+
+enum Network {
+  ARBITRUM
+  AURORA
+  AVALANCHE
+  BSC
+  CELO
+  CRONOS
+  ETHEREUM
+  FANTOM
+  HARMONY
+  MOONBEAM
+  MOONRIVER
+  OPTIMISM
+  POLYGON
+  XDAI
+}
+
+enum ProtocolType {
+  EXCHANGE
+  LENDING
+  YIELD
+  BRIDGE
+  GENERIC
+  # Will add more
+}
+
+type Token @entity {
+  " Smart contract address of the market "
+  id: ID!
+
+  " Mirrored from the smart contract "
+  name: String!
+  symbol: String!
+  decimals: Int!
+
+  ###### internal use ########
+  # used for other stats like marketcap
+  totalSupply: BigDecimal!
+
+  # token specific volume
+  tradeVolume: BigDecimal!
+  tradeVolumeUSD: BigDecimal!
+  untrackedVolumeUSD: BigDecimal!
+  # transactions across all pairs
+  txCount: BigInt!
+
+  # liquidity across all pairs
+  totalLiquidity: BigDecimal!
+
+  # derived prices
+  derivedBNB: BigDecimal!
+}
+
+enum RewardTokenType {
+  DEPOSIT
+  BORROW
+}
+
+type RewardToken @entity {
+  " Smart contract address of the market "
+  id: ID!
+
+  " Mirrored from the smart contract "
+  name: String!
+  symbol: String!
+  decimals: Int!
+
+  type: RewardTokenType!
+}
+
+#############################
+##### Protocol Metadata #####
+#############################
+
+interface Protocol {
+  " Smart contract address of the protocol's main contract (Factory, Registry, etc) "
+  id: ID!
+
+  " Name of the protocol, including version. e.g. Uniswap v3 "
+  name: String!
+  " Slug of protocol, including version. e.g. uniswap-v3 "
```

Comment:
can you format all the fields properly (spacing between fields)

Reply:
fixed